deploy-manage/remote-clusters/eck-remote-clusters-to-external.md

```diff
@@ -0,0 +1,355 @@
+---
+navigation_title: To an external cluster or deployment
+applies_to:
+  deployment:
+    eck: ga
+products:
+  - id: cloud-kubernetes
+sub:
+  local_type_generic: cluster
+  remote_type_generic: deployment or cluster
+---
+
+# Connect an ECK-managed cluster to an external cluster or deployment
+
+These steps describe how to configure a remote cluster connection from an {{es}} cluster managed by {{eck}} (ECK) to an external {{es}} cluster, where external refers to any cluster not managed by the same ECK operator. The remote cluster can be self-managed, part of an {{ech}} (ECH) or {{ece}} (ECE) deployment, or managed by a different ECK operator.
+
+Once the connection is established, you’ll be able to [run CCS queries from {{es}}](/solutions/search/cross-cluster-search.md) or [set up CCR](/deploy-manage/tools/cross-cluster-replication/set-up-cross-cluster-replication.md).
```

Comment:
always prefer `after` instead of `once` because `once` can be interpreted as "one time"

```suggestion
After the connection is established, you’ll be able to [run CCS queries from {{es}}](/solutions/search/cross-cluster-search.md) or [set up CCR](/deploy-manage/tools/cross-cluster-replication/set-up-cross-cluster-replication.md).
```